src/core/client/admin/routes/Community/UserRow.css

```diff
@@ -37,6 +33,5 @@
 .deleted {
   display: inline-block;
   padding-left: var(--spacing-2);
-
   color: var(--palette-warning-main);
```

Comment:
We can discuss this colouring with Sam so we can get an appropriate colour variable generated for it.